mig-operator-source.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 spec:
   type: appregistry
   endpoint: https://quay.io/cnr
-  registryNamespace: konveyor
+  registryNamespace: jmontleon
```

Comment:
Is this not `konveyor`?

Reply:
It should be. Thanks for catching. changed it to test and picked it up in the commit by accident.